### DISCUSSION TITLE
Updating "Blank" template to support Gallery

### DISCUSSION THREAD

**Discussion by johnwoltman** [GetPublii/Publii#1569] [Ideas]
Could the "Blank" start template be updated to have the basic support for Gallery items?  I haven't been able to figure out how to add the support in my own theme based on Blank.

**Comment by michaelfluegge** [comment]
@johnwoltman , do you still have the issue with galleries in the Blank theme? I can create a post with a gallery in it and the Lightbox plugin work with it, too.

**Reply by johnwoltman** [reply]
I will check.
